devel/uboot-tools/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                uboot-tools
+github.setup        u-boot u-boot 2020.07 v
+revision            0
+github.tarball_from archive
+license             GPL-2
+categories          devel
+platforms           darwin
+maintainers         {@rubendibattista gmail.com:rubendibattista} openmaintainer
+description         Binary tools for U-Boot.
+long_description    U-Boot is a boot loader for Embedded boards based on \
+                    PowerPC, ARM, MIPS and several other processors. This \
+                    port compiles some binary tools that can be used to \
+                    deploy u-boot on embedded devices.
+
+checksums           rmd160  124de3a5355915ccb2291b4a01f21e7710290521 \
+                    sha256  616b446e15d1cd1ab6461ebb61ac6655a2b13e902fe0601f36c4affb3949d416 \
+                    size    19630685
+
+depends_build       port:gmake
+
+depends_lib         port:openssl
+
+configure.cmd       gmake defconfig
+configure.pre_args
+
+build.target        tools
+build.cmd           gmake
+build.args-append   KBUILD_HOSTCFLAGS=-I${prefix}/include
+
+destroot {
+    set bindir ${destroot}${prefix}/bin
+    set docdir ${destroot}${prefix}/share/man/man1
+
+    xinstall -d ${bindir}
+    xinstall -d ${docdir}
+
+    xinstall -m 755 -W ${worksrcpath}/tools mkimage dumpimage ${bindir}
+    
+    # Compress man file
+    set gzip [findBinary gzip ${portutil::autoconf::gzip_path}]
+    system "$gzip ${worksrcpath}/doc/mkimage.1"
```

Comment:
Don't compress the manpage yourself. Install the uncompressed manpage. MacPorts will compress all manpages for you.